settings.py

```diff
@@ -1055,15 +1049,13 @@
 db_settings['PORT'] = db_settings.get('PORT', '5432')
 options = db_settings.get('OPTIONS')
 db_settings['OPTIONS'] = '?{}'.format(urlencode(options)) if options else ''
-
-if UNIT_TESTING and sys.argv[1] == "test":
-    # Use test database name, but only if running the test command.
-    # Django uses different database names than the ones in DATABASES
-    # when setting up for tests. However, UNIT_TESTING may be true in
-    # some cases where django is not running tests (js tests on travis),
-    # and therefore does not change the database name.
-    from django.db.backends.creation import TEST_DATABASE_PREFIX
-    db_settings['NAME'] = TEST_DATABASE_PREFIX + db_settings['NAME']
+# Use test database name, but only if running the test command.
+# Django uses different database names than the ones in DATABASES
+# when setting up for tests. However, UNIT_TESTING may be true in
+# some cases where django is not running tests (js tests on travis),
+# and therefore does not change the database name.
+db_settings['NAME'] = helper.get_db_name(db_settings['NAME'],
+                                         UNIT_TESTING and helper.is_testing())
```

Comment:
is it necessary to AND `UNIT_TESTING` and `helper.is_testing()` since `UNIT_TESTING` has been assigned the return value from `helper.is_testing()` higher up in the file?